src/Cart.Service/Cart.Service/Endpoints/CartEndpoints.cs

```diff
@@ -9,68 +9,67 @@ public static class CartEndpoints
 
     public static WebApplication RegisterCartEndpoints(this WebApplication app)
     {
-        var cartApi = app.MapGroup("/api");
+        var versionOne = new Asp.Versioning.ApiVersion(1, 0);
+        var currentVersion = new Asp.Versioning.ApiVersion(Program.CurrentVersion.Major, Program.CurrentVersion.Minor);

```

Comment:
This is not covered with requirements, but I would not suggest couple api versioning and assembly version.
In your scenario when you want add endpoints with 3 rd version, you will need to update all routes for 2nd veriosn from 'currnetVersion' to versionTwo